frontend/src/components/AmountInfoCard/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { ReactNode } from "react";
+import { Card, Heading5 } from "@stellar/design-system";
+
+import "./styles.scss";
+
+type Props = {
+  title: string;
+  titleIcon?: ReactNode;
+  amount: string | number;
+  amountBy?: string | number;
+  amountPrefixContent?: ReactNode;
+  description?: string;
+};
+
+export const AmountInfoCard = ({
+  title,
+  titleIcon,
+  amount,
+  amountBy,
+  amountPrefixContent,
+  description,
+}: Props) => (
+  <div className="AmountInfoCard">
+    <Card noShadow>
+      <div className="AmountInfoCard__card_title">
+        {titleIcon}
+        <div className="AmountInfoCard__card_title__title">{title}</div>
+      </div>
+      <div className="AmountInfoCard__content">
+        {amountPrefixContent}
+        <Heading5>
+          {amount}
+          {amountBy && (
+            <span className="AmountInfoCard__content__amount_by">
+              {amountBy}
+            </span>
+          )}
```

Comment:
Just to be safe, we prefer to use a conditional operator in cases like this (`amountBy ? <span /> : null`). Even though there should be no issues in this specific scenario, it's good to stay consistent. Same for `description` below.

Reply:
I agree with you. In these cases, I usually add a "double negative" before the check expression so that it always will be converted to a boolean value, which in JSX renders nothing, meaning that we keep consistency. What do you think about this approach? It would be something like:
```suggestion
          {!!amountBy && (
            <span className="AmountInfoCard__content__amount_by">
              {amountBy}
            </span>
          )}
```

Reply:
Sure, that works. :+1: